smoker/server/daemon.py

```diff
@@ -263,10 +258,6 @@ def _shutdown(self, signum=None, frame=None, exitcode=0, exception=False):
 
         lg.info("Shutting down")
         try:
```

Comment:
No nice way. It's possible to stop the server, but it has to be done from a scope of a Request object, which means creating an API resource for that. I really dislike this way http://flask.pocoo.org/snippets/67/
There's a workaround, you can start the http server in a subprocess and terminate it later. I can add this if you think it's worth it.